ocfweb/docs/docs/staff/policies/keycard.md

```diff
@@ -0,0 +1,54 @@
+[[!meta title="Keycard policy"]]
+
+<!-- This policy was promulgated by the GMs on 8/18/2017. -->
```

Comment:
iso8601 pls

Reply:
Actually I'll spell out the date instead to harmonize it with the other policies.